Nightmare/Locales/ruRU.lua

```diff
@@ -1,23 +1,23 @@
 local L = BigWigs:NewBossLocale("Cenarius", "ruRU")
 if not L then return end
 if L then
-	--L.forces = "Forces"
-	--L.bramblesSay = "Brambles near %s"
+	L.forces = "Силы кошмара"
+	L.bramblesSay = "Колючки рядом с %s"
 end
 
 L = BigWigs:NewBossLocale("Elerethe Renferal", "ruRU")
 if L then
-	--L.isLinkedWith = "%s is linked with %s"
-	--L.yourLink = "You are linked with %s"
-	--L.yourLinkShort = "Linked with %s"
+	L.isLinkedWith = "%s связан(а) с %s"
+	L.yourLink = "Вы связаны с %s"
+	L.yourLinkShort = "Связаны с %s"
 end
 
 L = BigWigs:NewBossLocale("Il'gynoth", "ruRU")
 if L then
-	--L.custom_off_deathglare_marker = "Deathglare Tentacle marker"
-	--L.custom_off_deathglare_marker_desc = "Mark Deathglare Tentacles with {rt6}{rt5}{rt4}{rt3}, requires promoted or leader.\n|cFFFF0000Only 1 person in the raid should have this enabled to prevent marking conflicts.|r\n|cFFADFF2FTIP: If the raid has chosen you to turn this on, having nameplates enabled or quickly mousing over the spears is the fastest way to mark them.|r"
+	L.custom_off_deathglare_marker = "Маркировка Щупалец смертоносного взгляда"
+	L.custom_off_deathglare_marker_desc = "Помечать Щупальца смертоносного взгляда метками {rt6}{rt5}{rt4}{rt3}, требуется быть помощником или лидером рейда.\n|cFFFF0000Только один человек в рейде должен включить данную опцию чтобы избежать конфликтов.|r\n|cFFADFF2FПодсказка: Проведите мышкой над щупальцами, чтобы быстрее отметить их.|r"
 
-	--L.bloods_remaining = "%d |4Blood:Bloods; remaining"
+	L.bloods_remaining = "%d |4Кровь:Крови; осталось" --huh?

```

Comment:
You don't have to make it singular/plural, that's completely optional. You can replace it with just one word if you wish.